### ISSUE TITLE
Ability to modify swagger strings

### ISSUE THREAD

**Issue by here0to0learn** [kennethreitz/responder#242]
The built-in openapi support is great! Kudos to that. However, it would be nice if there were ways to modify more swagger strings such as page title, `default`, `description` etc. Not a very important feature but would be nice to have to make swagger docs more customizable. 

I am thinking passing common variables within `responder.API` along with already existing "title" and "version"?

**Comment by iancleary** [comment]
Hello,

@here0to0learn I ran into this issue last weekend while working with responder. I came to a similar idea with passing variable to the api, similar to how title and version work.  I was thinking of taking a spin at this issue this week to learn more about.  If you already have some coded, let me know and I will let you make a pull request.  Otherwise, I'll let you work on it.

@kenneth-reitz @taoufik07 Does this sound like a "good first issue" and/or upforgrabs?

Thanks,
-Ian

**Comment by iancleary** [comment]
I think this issue can stay open as we implement other customization of the swagger docs.  The "default" text about route groups, for example.

That said, https://github.com/kennethreitz/responder/pull/306 implemented the OpenAPI Info Object.

**Comment by iancleary** [comment]
@here0to0learn The "default" text is a default tag of every route and method combo, if left unspecified.

You can modify your class's swagger string like the following to use this right now:

An example below:

~~~~
class ObjectView(OpenBaseView):
    """An endpoint for Adding Objects
        ---
        post:
            summary: Create a object and add it to the database
            description: Create a object in the database
            tags:
                - objects
            responses:
                200:
                    description: Successful
                    content:
                        application/json:
                            schema:
                                $ref: '#/components/schemas/Object_POST'
        get:
            summary: Read the entire list of objects in the database
            description: Read the list of objects in the database
            tags:
                - objects
            responses:
                200:
                    description: Successful read of all objects in the database
~~~~

The rest of your request was implemented by #306.

Using responder==1.3.0 (see [https://python-responder.org/en/latest/tour.html#interactive-documentation](https://python-responder.org/en/latest/tour.html#interactive-documentation)), here is an example with the above tags:

![Screenshot from 2019-03-10 19-43-24](https://user-images.githubusercontent.com/9398410/54097420-43d46500-436d-11e9-9cc8-f8dabe04157f.png)

@taoufik07, I would vote for this issue to be closed as I believe everything in @here0to0learn's original issue has been implemented. 

Thanks,
-Ian
